packages/website/builder/config.json

```diff
@@ -40,10 +40,25 @@
       "../../../docs/api/action",
       "../../../docs/api/module",
       "../../../docs/api/providers",
-      "../../../docs/api/components",
       "../../../docs/api/compute",
       "../../../docs/api/universalcontroller"
     ],
+    "views": [{
+      "name": "react",
+      "path": "../../node_modules/@cerebral/react/README"
+    }, {
+      "name": "vue",
+      "path": "../../node_modules/@cerebral/vue/README"
+    }, {
+      "name": "inferno",
+      "path": "../../node_modules/@cerebral/inferno/README"
+    }, {
+      "name": "preact",
+      "path": "../../node_modules/@cerebral/preact/README"
+    }, {
+      "name": "angularjs",
+      "path": "../../node_modules/@cerebral/angularjs/README"
+    }],
```

Comment:
We need to keep these sorted to help with navigation... Order is:

```
angularjs
inferno
preact
react
vue
```

Reply:
Agreed, updated :)